pkg/metrics/metrics.go

```diff
@@ -71,41 +72,58 @@ func NewServerMetrics() *ServerMetrics {
 				},
 				[]string{scheduleLabel},
 			),
+			backupDurationSeconds: prometheus.NewHistogramVec(
+				prometheus.HistogramOpts{
+					Namespace: metricNamespace,
+					Name:      backupDurationSeconds,
+					Help:      "Time taken to complete backup, in seconds",
+					// Use 10 exponential buckets starting at 30s
+					Buckets: prometheus.ExponentialBuckets(30.0, 3, 10),
```

Comment:
This would create the following buckets, correct?

1. 30s
1. 1m30s
1. 4m30s
1. 13m30s
1. 40m30s
1. 2h1m30s
1. 6h4m30s
1. 18h13m30s
1. 54h40m30s
1. 164h1m30s

Reply:
Yeah - I think 9 & 10 might be too high, perhaps even 8.

I think Linear buckets of 1 minute may leave out the high end if we only have 10, but I don't know if there's any sort of penalty for making many buckets that go unused.